tox.ini

```diff
@@ -6,8 +6,8 @@ envlist=
 
 [testenv]
 deps=
-  mock
-  requests-mock
+	google-compute-engine
```

Comment:
boto was complaining on Travis 
```
Traceback:
tests/test_to_s3.py:5: in <module>
    from moto import mock_s3
.tox/py/lib/python3.6/site-packages/moto/__init__.py:9: in <module>
    from .autoscaling import mock_autoscaling, mock_autoscaling_deprecated  # flake8: noqa
.tox/py/lib/python3.6/site-packages/moto/autoscaling/__init__.py:2: in <module>
    from .models import autoscaling_backends
.tox/py/lib/python3.6/site-packages/moto/autoscaling/models.py:2: in <module>
    from boto.ec2.blockdevicemapping import BlockDeviceType, BlockDeviceMapping
.tox/py/lib/python3.6/site-packages/boto/__init__.py:1216: in <module>
    boto.plugin.load_plugins(config)
.tox/py/lib/python3.6/site-packages/boto/plugin.py:93: in load_plugins
    _import_module(file)
.tox/py/lib/python3.6/site-packages/boto/plugin.py:75: in _import_module
    return imp.load_module(name, file, filename, data)
.tox/py/lib/python3.6/imp.py:234: in load_module
    return load_source(name, filename, file)
.tox/py/lib/python3.6/imp.py:172: in load_source
    module = _load(spec)
/usr/lib/python2.7/dist-packages/google_compute_engine/boto/compute_auth.py:19: in <module>
    from google_compute_engine import logger
E   ModuleNotFoundError: No module named 'google_compute_engine'
```
This answer summarises the reason https://github.com/GoogleCloudPlatform/compute-image-packages/issues/262#issuecomment-225736860